communitybsl.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.9.0,1.10.0
-publicVersions=1.11.0
+archivedVersions=1.9.0,1.10.0,1.11.0
+publicVersions=1.12.0
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.12.0.description=9 new rules, bug fixes, optimize
+1.12.0.sqVersions=[8.9,LATEST]
```

Comment:
It's time to begin starting ranges with the current LTS: 9.9

Reply:
done